lib/version_utils.pm

```diff
@@ -232,6 +233,14 @@ sub is_sle_micro {
     return check_var('DISTRI', 'sle-micro');
 }
 
+=head2 is_selfinstall
+
+Check if SLEM is in flavor of self installable iso
+=cut
+sub is_selfinstall {
+    return is_sle_micro && (get_var('FLAVOR') =~ /selfinstall/i);
```

Comment:
shall we make this more generic and omit `is_sle_micro` ?

Reply:
I would go generic, there is no other flavor like this so far and I don't think there are plans. Anyway, if there is something else besides sle micro, it's a valid function for it as well (e.g. microOS?)